### PR TITLE
Allow different scale transform for x and y coordinates

### PR DESCRIPTION
**Pull request by cixzhang**

### CODE REVIEW

File: build/grapher.js [hunk 1]
```diff
@@ -122,15 +122,15 @@ Grapher.prototype = {};
 /**
   * grapher.initialize
   * ------------------
-  *
```
Comment (curt-mitch) [inline build/grapher.js]: Are all of these empty spaces auto-generated?

File: modules/renderers/renderer.js [hunk 2]
```diff
@@ -11,19 +11,22 @@
...
+      this.setScale(o.scale);
+      this.setTranslate(o.translate);
 
       this.resize();
     },
     setNodes: function (nodes) { this.nodeObjects = nodes; },
     setLinks: function (links) { this.linkObjects = links; },
-    setScale: function (scale) { this.scale = scale; },
+    setScale: function (scale) {
+      if (!(scale instanceof Array)) scale = [scale, scale];
```
Comment (curt-mitch) [inline modules/renderers/renderer.js]: Isn't there an isArray utility method?

Comment (cixzhang) [inline modules/renderers/renderer.js]: Yeah. Changing this!

**Comment by curt-mitch** [comment]
Looks good! 👍

**Pull request merged**